packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2789,7 +2795,34 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
       onLiveTextInput: liveTextInputEnabled
           ? () => _startLiveTextInput(SelectionChangedCause.toolbar)
           : null,
-    );
+    )..addAll(_textProcessingActionButtonItems);
```

Comment:
Are all text processing toolbar buttons always added to the end of the menu?

Reply:
Interesting questions! Looking at the Android native code, there are no way change the order.

The Android menu is initialized here:
https://android.googlesource.com/platform/frameworks/base/+/refs/heads/main/core/java/android/widget/Editor.java#4642

And the customs actions are added to the menu here:
https://android.googlesource.com/platform/frameworks/base/+/refs/heads/main/core/java/android/widget/Editor.java#7943

And the menu construction is done here:
https://android.googlesource.com/platform/frameworks/base/+/refs/heads/main/core/java/android/widget/Editor.java#4608

From this link we can see that `populateMenuWithItems(menu)` is always called before `mProcessTextIntentActionsHandler.onInitializeMenu(menu)`.